README.md

```diff
@@ -178,3 +178,14 @@ If you use FairScale in your publication, please cite it by using the following
   year =         {2021}
 }
 ```
+
+## FAQ
```

Comment:
Should we add a whats new section or a Note section? This FAQ is at the bottom right?

Reply:
ok. I think we should continue updating "what new section" as the pytorch/fairseq README.md from now on.